src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/SearchDAO.scala

```diff
@@ -14,5 +14,5 @@ trait SearchDAO extends LazyLogging {
   def bulkIndex(docs: Seq[Document]): Unit
   def indexDocument(doc: Document): Unit
   def deleteDocument(id: String): Unit
-
+  def findDocuments(term: String, from: Int = 0, size: Int = 10): LibrarySearchResponse
```

Comment:
This signature is going to have to change as soon as we support facets - we'll need more criteria than a String. I think we'll need to create some kind of `SearchCriteria` object, holding the search term, from/size for pagination, sort criteria, and (soon) facet selections, eventually things like sort. I'm fine leaving this PR as-is, but if you feel like tackling that refactor, all good.

EDIT: oh wait, you have a `LibrarySearchParams` model class already! Why not have this method accept `LibrarySearchParams` instead? Then you could also handle the default values for `from` and `size` in the model, not here; you could address Sarah's comment about accepting `Option[String]` for the search term in the model too.